src/main/java/org/fundacion/jala/converter/view/metadata/OutputInfo.java

```diff
@@ -5,8 +5,9 @@
  * ("Confidential Information"). You shall not disclose such Confidential
  * Information and shall use it only in accordance with the terms of the
  * license agreement you entered into with Fundacion Jala
+ *
+ * @author Paola Ximena Aguilar Quiñonez
```

Comment:
author second last name is Quiñones with s

Reply:
Have changed all the files in this package to Quiñonez with S